README.md

```diff
@@ -18,7 +18,76 @@
+| **Schematics**                                | ⚠️    | -                                                                                                               | ✅    | -                                                                             | ✅    | -                                                                                                         |                                                                                                                             |
+| - Generate Application                        | ⚠️    | -                                                                                                               | ✅    | `ng generate app`                                                             | ✅    | `ng generate app`                                                                                         |                                                                                                                             |
+| - Serve Application                           | ✅    | -                                                                                                               | ✅    | `ng serve`                                                                    | ✅    | `ng serve`                                                                                                |                                                                                                                             |
+| - Build Application                           | ✅    | -                                                                                                               | ✅    | `ng build`                                                                    | ✅    | `ng build`                                                                                                |                                                                                                                             |
+| **Angular Plugin**                            | ✅    | [@ng-rspack/plugin-angular](https://github.com/Coly010/ng-rspack-build/tree/main/packages/build-plugin-angular) | ✅    | [@angular/build](https://github.com/angular/angular/tree/main/packages/build) | ✅    | [@angular-devkit/build-angular](https://github.com/angular/angular-cli/tree/main/packages/angular_devkit) |                                                                                                                             |
```

Comment:
I don’t think this row makes sense or is needed

Reply:
Don't we have a serve and dev feature?

Reply:
Oh the suggestion included too much.

The intended line is the Angular Plugin line

Reply:
removed